### ISSUE TITLE
Update rio dependencies

### ISSUE THREAD

**Issue by damooo** [pchampin/sophia_rs#124]
Currently rio's crates are updated to 0.7.x minor, and sophia pegs to 0.6.x. Thus one have to peg rio to older version to resolve clash.

**Comment by pchampin** [comment]
yes, an new release of Sophia should be released soon (it is currently stalled on the stabilization of Generic Associated Types), that should fix that.
